### ISSUE TITLE
Refresh files list by clicking on breadcrumb entry

### ISSUE THREAD

**Issue by heinrichmartin** [nextcloud/server#7958]
### Steps to reproduce
1. Open a folder in the files app
2. Add a file by by other means (android app, webDAV)
3. Wait for the new file to appear in the browser app (Is auto-refresh implemented? If yes, it's a bug here ...)
4. Click on the current folder in the breadcrumbs
5. Refresh the page or navigate up and back to the folder

### Expected behaviour
4. The file list is updated to show the files on the server, including the new one.

### Actual behaviour
4. Nothing happens when clicking on the current folder in the breadcrumbs.
For all ancestor folders a click changes to that folder.
5. is a workaround.

### Server configuration

**Operating system**:
Ubuntu 16.04.3 LTS

**Web server:**
Apache/2.4.18 (Ubuntu)

**Database:**
mysql 10.0.33

**PHP version:**
7.0.22-0ubuntu0.16.04.1

**Nextcloud version:** (see Nextcloud admin page)
12.0.4

**Updated from an older Nextcloud/ownCloud or fresh install:**
quite a few versions already, skipped none

**Where did you install Nextcloud from:**
probably not of interest for this issue

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - activity: 2.5.2
  - announcementcenter: 3.1.1
  - apporder: 0.4.1
  - bruteforcesettings: 1.0.3
  - calendar: 1.5.7
  - comments: 1.2.0
  - contacts: 2.0.1
  - dav: 1.3.0
  - federatedfilesharing: 1.2.0
  - files: 1.7.2
  - files_markdown: 2.0.1
  - files_pdfviewer: 1.1.1
  - files_sharing: 1.4.0
  - files_texteditor: 2.4.1
  - files_trashbin: 1.2.0
  - files_versions: 1.5.0
  - files_videoplayer: 1.1.0
  - firstrunwizard: 2.1
  - gallery: 17.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.0.0
  - nextcloud_announcements: 1.1
  - notes: 2.3.2
  - notifications: 2.0.0
  - oauth2: 1.0.5
  - password_policy: 1.2.2
  - provisioning_api: 1.2.0
  - quota_warning: 1.1.1
  - serverinfo: 1.2.0
  - survey_client: 1.0.0
  - systemtags: 1.2.0
  - theming: 1.3.0
  - twofactor_backupcodes: 1.1.1
  - updatenotification: 1.2.0
  - workflowengine: 1.2.0
Disabled:
  - admin_audit
  - encryption
  - federation
  - files_external
  - sharebymail
  - user_external
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***FAILED TO REMOVE SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***FAILED TO REMOVE SENSITIVE VALUE***"
        ],
        "datadirectory": "\/var\/www\/nextcloud\/data",
        "overwrite.cli.url": "***FAILED TO REMOVE SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "12.0.4.3",
        "dbname": "nextcloud",
        "dbhost": "localhost",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "mail_from_address": "noreply",
        "mail_smtpmode": "smtp",
        "mail_domain": "***FAILED TO REMOVE SENSITIVE VALUE***",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "\/var\/run\/redis\/redis.sock",
            "port": 0
        },
        "mail_smtpsecure": "tls",
        "mail_smtpauth": 1,
        "mail_smtphost": "***FAILED TO REMOVE SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "theme": "",
        "loglevel": 1,
        "defaultapp": "apporder",
        "singleuser": true
    }
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
WebDAV, android app

### Client configuration
**Browser:**
Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:57.0) Gecko/20100101 Firefox/57.0

**Operating system:**
Windows

**Comment by heinrichmartin** [comment]
Actually, this is not a duplicate (at least not of the linked issue) - auto-refresh was just a side-note:
### Steps to reproduce
4. Click on the current folder in the breadcrumbs
### Expected behaviour
4. The file list is updated to show the files on the server, including the new one.

**Comment by MorrisJobke** [comment]
Duplicate of #2332

**Comment by MorrisJobke** [comment]
Fixed by #6537 in 13.
